scripts/check-origins

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
```

Comment:
This file is almost identical to the file `scripts/package`,
how about moving the shared parts into another script file? 

Another reason for the suggestion is that today we need to change the default value of `SYSTEM_CHART_DEFAULT_BRANCH` between `dev-v2.x` and `release-v2.x` in the file `scripts/package` before/after releasing rancher.  If we add this file, it means one more place for us to change the same variable.

Reply:
I'll move the common code into a shared script, though the `check-origins` script is expected to only be used in a developer workflow, it doesn't contribute to a release or produce any artifacts on its own so I wasn't anticipating needing to update it.

Reply:
A new file, `package-env`, was created to handle the common environment variables between the two scripts